flutter/test/load_contexts_integrations_test.dart

```diff
@@ -0,0 +1,143 @@
+import 'package:flutter/services.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:sentry/sentry.dart';
```

Comment:
`sentry_flutter.dart` already imports this

Reply:
✅